cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -67,9 +67,17 @@ else
     VSIM_ARGS += -onfinish exit
 endif
 
+FLI_LIB := $(LIB_DIR)/libcocotbfli_modelsim.$(LIB_EXT)
+# if this target is run, then cocotb did not build the library
+$(FLI_LIB):
+	@echo -e "ERROR: cocotb was not installed with an FLI library, as the mti.h header could not be located. \n\
+	If you installed an fli-capable simulator after cocotb, you will need to reinstall cocotb. \n\
```

Comment:
```suggestion
	@echo -e "ERROR: cocotb was not installed with an FLI library, as the mti.h header could not be located.\n\
	If you installed an fli-capable simulator after cocotb, you will need to reinstall cocotb.\n\
```
Let's keep trailing spaces out of our output, since we also keep them out of our code.